packages/calling/src/CallingClient/calling/call.test.ts

```diff
@@ -717,32 +718,6 @@ describe('Call Tests', () => {
       {file: 'call', method: 'updateMedia'}
     );
   });
-
-  describe('#addSessionConnection', () => {
```

Comment:
We do not require these tests anymore as it is already being validated in the media core repo.